### DISCUSSION TITLE
Darkmode

### DISCUSSION THREAD

**Discussion by ecospecifier** [vikramlearning/blazorbootstrap#466] [General]
With bootstrap 5.3 will we see Blazor Bootstrap embrace darkmode firstclass? I notce a lot of components like eg. toasts don't respect the darkmode class on the body of the parent.

**Comment by gvreddy04** [comment]
@ecospecifier Please create an issue for this. We will add the Dark mode support in the future releases.

**Reply by mikelekyr** [reply]
Dear team, I would like to ask you how is it now with the dark mode. Thank you

**Reply by gvreddy04** [reply]
@mikelekyr Thank you for using BlazorBootstrap. The dark mode feature is currently under development.
